x-pack/metricbeat/module/azure/monitor_service.go

```diff
@@ -124,3 +124,24 @@ func (service *MonitorService) GetMetricValues(resourceId string, namespace stri
 	}
 	return metrics, interval, nil
 }
+
+// getResourceNameFormId maps resource group from resource ID
+func getResourceNameFromId(path string) string {
```

Comment:
Maybe these functions could be moved to a `utils.go` file since look too generic?